_includes/releases/v23.1/v23.1.0-beta.3.md

```diff
@@ -0,0 +1,63 @@
+- Introduced a "draining node" **Node Status** on the [Cluster Overview panel](../v23.1/ui-cluster-overview-page.html#cluster-overview-panel) so that "draining node" has it own value instead of counting as a "dead node". [#101794][#101794]
+- Added a time scale selector to the [**Diagnostics** tab](../v23.1/ui-statements-page.html#diagnostics) under the Statement Details page. This allows users to view bundles from the selected period only. [#101803][#101803]
+
+<h3 id="v23-1-0-beta-3-bug-fixes">Bug fixes</h3>
+
+- Fixed a bug that caused internal errors when executing [user-defined functions](../v23.1/user-defined-functions.html) with empty bodies. This bug was only present in alpha pre-release versions of 23.1. [#101383][#101383]
+- Fixed a bug that caused a [restore](../v23.1/restore.html) to fail occasionally due to incorrect schema ID resolution when restoring a [backup](../v23.1/backup.html) with user-defined schemas. [#101310][#101310]
+- Fixed a bug in parsing a [`search_path`](../v23.1/sql-name-resolution.html#current-schema) with a quote in it when specified in the connection string. Also, the `search_path` session variable now supports schema names that have commas in them. [#101492][#101492]
+- Fixed a bug that has existed since [user-defined functions](../v23.1/user-defined-functions.html) were introduced that could cause a function call to resolve to the wrong function after changes to the schema search path. [#101513][#101513]
+- Previously, CockroachDB v23.1 alpha and beta versions would panic on `start` command when the `GOMEMLIMIT` environment variable was set and the `--max-go-memory` flag wasn't specified. This is now fixed. [#101565][#101565]
+- Fixed a bug that caused errors in test builds and potentially incorrect results in release builds when invoking a [user-defined function](../v23.1/user-defined-functions.html) with a subquery argument. This bug was only present in v23.1 alpha versions. [#101639][#101639]
+- Fixed an internal error that could occur when the `enforce_home_region` [session setting](../v23.1/set-vars.html) is on and the input to the lookup join is a `SELECT` of [scalar expressions](../v23.1/scalar-expressions.html) (e.g., `1+1`). Also, subqueries with no home region now error out with `enforce_home_region` set. [#101504][#101504]
+- Point inserts and updates that write to a remote region of a table create with the [`REGIONAL BY ROW AS`](../v23.1/create-table.html#create-a-table-with-a-regional-by-row-locality-using-a-custom-region-column) clause will now error out. [#101708][#101708]
+- Fixed a bug in the [built-in functions](../v23.1/functions-and-operators.html) `pg_get_indexdef` and `col_description` that could cause the functions to return errors if the user created tables named `pg_indexes` or `pg_attribute`. Or, if the user created a schema named `system` with a table named `comments`. This bug was only present in pre-release v23.1. [#101690][#101690]
```

Comment:
I think this is better as "pre-release versions of v23.1"

meta: I like that phrasing better than saying alphas/betas/etc, but it doesn't really matter I guess